fundraiser/src/utils.sw

```diff
@@ -0,0 +1,18 @@
+library utils;
+
+dep errors;
+
+use std::{assert::require, chain::auth::{AuthError, msg_sender}, identity::Identity, result::*};
+
+use errors::UserError;
+
+pub fn sender_identity() -> Identity {
+    // TODO: use msg_sender().unwrap() (and remove this function) when that becomes functional
+    // Related: https://github.com/FuelLabs/sway/pull/1958
+    let sender: Result<Identity, AuthError> = msg_sender();
+    sender.unwrap()
```

Comment:
Weren't you just saying that this should be able to be chained, without the temp variable `sender` ?

Reply:
I think this may have been just before the release which fixed this